src/browser/atom-portable.coffee

```diff
@@ -4,31 +4,34 @@ ipc = require 'ipc'
 
 module.exports =
 class AtomPortable
-  @getPortableAtomHomePath: ->
+  @getPortableAtomHomePath: (platform) ->
+    # Mac has a deeper path for the executable in the application package, so need a different
```

Comment:
@thomasjo That's a good point and can change this comment, but not very consistent with references elsewhere.  There's more "Mac" than "OS X" in the project (and also some MacOS and OSX).